### DISCUSSION TITLE
Set zones off/manual/scheduled

### DISCUSSION THREAD

**Discussion by andr3w-it** [wrongisthenewright/ebusd-configuration-ariston-bridgenet#21] [Q&A]
Hello,
many thanks in advance for your great work!
I have an Aristom Nimbus 110 M-T and I'm integrating it with OpenHAB and also with Vimar By-Me system, using your config file with ebusd. I integrated some great features in both systems; now I'm trying to integrate the zones management.
I tried to found how to set the zones in off/manual/scheduled mode, but I didn't find the right lines.
Could you please let me know if you have decoded it?
I tried to use the "z1_heat_request_mode" parameter for zone 1, but I think it isn't correct, because it modify the scheduling operation, ignoring (or not) the time windows and it doesn't set the zone in off/manual/scheduled mode.
Please, let me know.
Have a nice day!

Andrea

**Comment by wrongisthenewright** [comment]
Hi Andrea, I'm really sorry for this late reply, I simply missed your question....
I haven't the zone manager in my system so for me it will be difficult to help. For what Ive seen from others the zone manager is bus connected so it has it's own bus addresses.

You could help supplying these logs:

- output of ebuctl info command (if possible turn off the zone manager an hour or so before issuing the command, then turn it on and launch another ebusctl info) this should give us the zone manager addresses, missing from the first run but present on the second
- output of ebusctl grab result all
- then modify some zone manager related settings alter them then reset them to the desired value
- then run another ebusctl grab result all (this should show us the messages sent fromt he Sensys to the zone manager)

Sorry again for the delay

**Comment by andr3w-it** [comment]
Don't worry for the delay; I'm so busy too!

Regarding the question: I don't have the zone manager; the 110-M-T manages three zones on-board. I configured two zones that use a classic ON/OFF thermostat (not smart and not the Ariston one).Actually, I'm using only one of the two; so, the zone that is changing between launching the requested commands is only the zone 1.

Here the output of the ebusctl info when Zone 1 and Zone 2 are both OFF:
version: ebusd 23.3.23.3                                                         
update check: version 24.1 available, device firmware 1[5210] available          
device: 192.168.178.136:9999, TCP, enhanced, firmware 1.1[3501].1[3501]          
signal: acquired                                                                 
symbol rate: 75                                                                  
max symbol rate: 213                                                             
scan: finished                                                                   
reconnects: 0                                                                    
masters: 6                                                                       
messages: 348                                                                    
conditional: 0                                                                   
poll: 0                                                                          
update: 56                                                                       
address 00: master #1                                                            
address 03: master #11                                                           
address 10: master #2                                                            
address 13: master #12                                                           
address 18: slave #12, scanning                                                  
address 1e: slave, scanning                                                      
address 31: master #8, ebusd                                                     
address 36: slave #8, ebusd                                                      
address 7f: master #24                                                           
address 84: slave #24, scanning

Here the output of the ebusctl info when zone 1 is in manual mode (and zone 2 is OFF):
version: ebusd 23.3.23.3                                                         
update check: version 24.1 available, device firmware 1[5210] available          
device: 192.168.178.136:9999, TCP, enhanced, firmware 1.1[3501].1[3501]          
signal: acquired                                                                 
symbol rate: 118                                                                 
max symbol rate: 213                                                             
scan: finished                                                                   
reconnects: 0                                                                    
masters: 6                                                                       
messages: 348                                                                    
conditional: 0                                                                   
poll: 0                                                                          
update: 56                                                                       
address 00: master #1                                                            
address 03: master #11                                                           
address 10: master #2                                                            
address 13: master #12                                                           
address 18: slave #12, scanning                                                  
address 1e: slave, scanning                                                      
address 31: master #8, ebusd                                                     
address 36: slave #8, ebusd                                                      
address 7f: master #24                                                           
address 84: slave #24, scanning 

In attachment, the output of ebusctl grab result all when:
1 - Z1 OFF - Z2 OFF
2 - Z1 Manual - Z2 OFF
3 - Z1 Scheduled - Z2 OFF

I compared the output and I saw some differences, but I don't have your level of understanding of those parameters.
Please, let me know if you need anything.
Thank you in advance,

Andrea

[20250312_Commands.zip](https://github.com/user-attachments/files/19208219/20250312_Commands.zip)

**Comment by FrigerioSergio** [comment]
Don't know if you finally solve your problem.  

my setting for the 3 zone heating mode:
r,sensys,z3_heat_mode    ,Heat mode      ,13,fe,200f,c679,Z1,s,heat_mode,0=off;2=man;3=prog,,,,s,IGN:3,,,,Z2,s,heat_mode,0=off;2=man;3=prog,,,,s,IGN:3,,,,Z3,s,heat_mode,0=off;2=man;3=prog

this is the message on the ebusd that write the z3:

<01fe202003c67b006a   --> **00**6a Off
<01fe202003c67b0268   --> **02**68 man
<01fe202003c67b0369   --> **00**69 prog (schedule)

c679 =z1
c67a =z2
c67b =z3


r,Ebus_comandi,Z3_heat_mode,Z3 risc,,fe,2020,c67b,,s,BCD,0=off;2=man;3=prog
w,Ebus_comandi,Z3_heat_mode,Z3 risc,,fe,2020,c67b,,s,heat_mode
r,Ebus_comandi,Z2_heat_mode,Z2 risc,,fe,2020,c67a,,s,BCD,0=off;2=man;3=prog
w,Ebus_comandi,Z2_heat_mode,Z2 risc,,fe,2020,c67a,,s,heat_mode

heat_mode,UCH,0=off;2=man;3=prog  inside template_csv

confirmed working with this script, e.g.

alias: ebusd/Ebus_comandi/Z2_heat_mode/set/off
action: mqtt.publish
metadata: {}
data:
  qos: "1"
  retain: false
  topic: ebusd/Ebus_comandi/Z2_heat_mode/set
  payload: "off"

**Reply by andr3w-it** [reply]
Hello,
I added your lines and it works!
I'll test it for a couple of days, but it seems ok! Many thanks
